### ISSUE TITLE
BLOCKED Use AMD for JavaScript

### ISSUE THREAD

**Issue by fulldecent** [fulldecent/moodle-local_plugin_template#10]
BLOCKER: waiting for Moodle best practices template (Will to do)

Due to errors in the moodle-docker setup we have to inject JavaScript straight into HTML at

https://github.com/fulldecent/moodle-local_enrollment_tokens/blob/4cf5d9f03335574ce49b6582130c50bd297d97d9/activate.php#L78-L149

Blocking issue: https://github.com/moodlehq/moodle-docker/issues/287

**Comment by marharyta-morozova** [comment]
If this issue is still active? Could you provide more details, please?

**Comment by fulldecent** [comment]
Yes, this is explained in notes here https://github.com/fulldecent/moodle-local_plugin_template/issues/11

**Comment by marharyta-morozova** [comment]
resolved here https://github.com/fulldecent/moodle-local_plugin_template/tree/main/amd
